configs/default/JHEF-JHEF405PRO.config

```diff
@@ -98,11 +119,15 @@ set battery_meter = ADC
 set ibata_scale = 170
 set beeper_inversion = ON
 set beeper_od = OFF
+set osd_vbat_pos = 14571
+set osd_ah_sbar_pos = 14542
+set osd_ah_pos = 14414
+set osd_current_pos = 14634
```

Comment:
Please remove as this should be user defined or added using manufacturer preset.